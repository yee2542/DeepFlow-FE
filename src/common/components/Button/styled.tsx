import { ButtonComponentProps } from 'Components/Button/types';
import styled, { css } from 'styled-components';

const colorMapping = (color?: ButtonComponentProps['color']) => {
  switch (color) {
    case 'secondary':
      return css`
        background-color: #fff;
        color: var(--green-color);
        border: 1px solid var(--green-color);
        &:hover {
          color: var(--light-green-color);
          border: 1px solid var(--light-green-color);
        }
      `;
    case 'danger':
      return css`
        background-color: var(--error-color);
        color: #fff;
        &:hover {
          background-color: var(--light-error-color);
        }
      `;
    case 'success':
      return css`
        background-color: var(--success-color);
        color: #fff;
        &:hover {
          background-color: var(--light-success-color);
        }
      `;
    case 'disable':
      return css`
        background-color: var(--gray-color);
        color: var(--dark-gray-color);
        border: 1px solid var(--dark-gray-color);
        &:hover {
          cursor: not-allowed;
        }
      `;
    // default -> primary
    default:
      return css`
        background-color: var(--green-color);
        color: #fff;
        &:hover {
          background-color: var(--light-green-color);
        }
      `;
  }
};

const sizeMapping = (size?: ButtonComponentProps['size']) => {
  switch (size) {
    case 'small':
      return css`
        font-size: var(--small-body);
        padding: 10px 16px;
      `;
    case 'large':
      return css`
        font-size: var(--h5);
        padding: 12px 24px;
      `;
    // default -> medium
    default:
      return css`
        font-size: var(--body);
        padding: 11px 20px;
      `;
  }
};

export const ButtonStyle = styled.button<ButtonComponentProps>`
  border-radius: 100px;
  border: 0px;
  ${({ color }) => colorMapping(color)};
  ${({ size }) => sizeMapping(size)};
  cursor: pointer;
  font-weight: bold;
  &:focus {
    outline: 0;
    box-shadow: none !important;
  }
`;
