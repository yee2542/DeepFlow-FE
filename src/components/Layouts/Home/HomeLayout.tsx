import React from 'react';
import HomeNavbar from './HomeNavbar';

const HomeLayout: React.FC = (props) => {
  const { children } = props;
  return (
    <div>
      <HomeNavbar />
      <div>{children}</div>
    </div>
  );
};

export default HomeLayout;
