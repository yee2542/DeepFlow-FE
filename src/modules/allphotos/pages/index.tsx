import React from 'react';
import AllPhotosLayout from 'Modules/allphotos/components/Layouts';

const AllPhotosPages: React.FC = () => {
  return <AllPhotosLayout>here</AllPhotosLayout>;
};

export default AllPhotosPages;
