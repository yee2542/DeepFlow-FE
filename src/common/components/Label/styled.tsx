import styled from 'styled-components';
import { Tag } from 'antd';

export const LabelStyle = styled(Tag)`
  border-radius: 100px;
  font-size: var(--small-body);
  padding: 4px 8px 4px 8px;
`;
