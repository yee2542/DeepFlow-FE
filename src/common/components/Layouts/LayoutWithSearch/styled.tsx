import styled, { css } from 'styled-components';
import { Layout } from 'antd';
const { Sider, Content } = Layout;

type ContentWithPaddingPropsType = {
  padding: boolean;
};

export const CanvasWrapper = styled.div`
  overflow: hidden;

  aside {
    width: 250px !important;
    max-width: 250px !important;
    min-width: 250px !important;
    flex: 0 0 250px !important;
  }

  .ant-layout-header {
    background-color: white;
    position: fixed;
    width: 100%;
    box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
    z-index: 999;
    /* margin-left: 10%; */
  }

  .ant-layout-content {
    background-color: white;
    overflow-y: scroll;
    width: 100%;

    @media (min-width: 1600px) {
      padding: 10% 5% 5% 10%;
    }
    @media (min-width: 1200px) and (max-width: 1599px) {
      padding: 10% 5% 5% 15%;
    }
    @media (min-width: 992px) and (max-width: 1199px) {
      padding: 10% 5% 5% 20%;
    }
    @media (max-width: 991px) {
      padding: 20% 10% 5% 10%;
    }
  }
`;

export const ContentWrapper = styled(Layout)`
  margin-left: 10%;

  @media (max-width: 992px) {
    margin-left: 0%;
  }
`;

export const SiderWrapper = styled(Sider)`
  background-color: #4425a7;
  padding: 5% 2% 0% 2%;
  overflow: hidden;
  z-index: 1000;
  left: 0;
  position: fixed;
  height: 100vh;

  @media (max-width: 992px) {
    display: none;
  }
`;

export const MenuWrapper = styled.div`
  margin-top: 30%;
  width: 100%;
`;

export const IconWrapper = styled.span`
  margin-right: 10%;
`;

export const MenuItem = styled.div<{ isSelecting: boolean }>`
  width: 100%;
  cursor: pointer;
  color: white;

  p:hover {
    padding: 1em 1em 1em 1em;
    border-radius: 0.5em;
    background-color: #48cdd8;
  }

  p {
    padding: 1em 1em 1em 1em;
    border-radius: 0.5em;
    background-color: transparent;
  }

  ${(props) =>
    props.isSelecting &&
    css`
      p {
        padding: 1em 1em 1em 1em;
        border-radius: 0.5em;
        background-color: white;
        color: #4425a7;
        font-weight: 600;
      }

      p:hover {
        padding: 1em 1em 1em 1em;
        border-radius: 0.5em;
        background-color: #9271fc;
      }
    `}
`;

export const WrapperImage = styled.section`
  max-width: 70rem;
  margin: 4rem auto;
  display: grid;
  grid-gap: 1em;
  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
  grid-auto-rows: 300px;
`;

export const UserWrapper = styled.div`
  cursor: default;

  .ant-avatar-circle {
    color: #fde3cf;
    background-color: #f56a00;
  }
  .ant-avatar-circle :hover {
    color: #f56a00;
    background-color: #fde3cf;
  }
`;

export const ContentWithPadding = styled(Content)<ContentWithPaddingPropsType>`
  ${(props) =>
    props.padding &&
    `
    padding: 5% 0% 5% 0%;
    margin-left: 5%;
  `}
`;
