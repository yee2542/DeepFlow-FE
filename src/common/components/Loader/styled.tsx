import styled from 'styled-components';

export const Loading = styled.div`
  text-align: center;
  .anticon {
    font-size: 24px;
    color: #4425a7;
  }
`;
