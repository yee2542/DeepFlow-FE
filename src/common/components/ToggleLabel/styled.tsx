import styled from 'styled-components';
import { Tag } from 'antd';
import { ToggleLabelWrapperProps } from './types';

export const ToggelLabelWrapper = styled(Tag)<ToggleLabelWrapperProps>`
  border-radius: 100px;
  font-size: var(--small-body);
  padding: 0em 1.25em 0em 1.25em;
  background-color: white;
  color: #5a36cc;
  border: 1px solid #5a36cc;
  ${(props) =>
    props.selected &&
    `background-color: #5A36CC;
    color: white;
    `}
  &:hover {
    background-color: #f8eafa;
    color: #5a36cc;
    cursor: pointer;
  }
`;
