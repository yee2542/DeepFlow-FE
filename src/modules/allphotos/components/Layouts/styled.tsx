import styled from 'styled-components';
import { Layout } from 'antd';
const { Sider } = Layout;

export const CanvasWrapper = styled.div`
  .ant-layout-header {
    background-color: white;
  }

  .ant-layout-content {
    height: 100vh;
    background-color: white;
  }
`;

export const SiderWrapper = styled(Sider)`
  background-color: #4425a7;
  width: 294px;
  padding: 2%;
`;

export const MenuWrapper = styled.div`
  margin-top: 20%;
  width: 100%;
`;

export const Menu = styled.div`
  margin-top: 20%;
  a:hover {
    padding: 10%;
    border-radius: 2em;
    background-color: #48cdd8;
  }
  a:active {
    padding: 10%;
    border-radius: 2em;
    background-color: #48cdd8;
    position: inherit;
  }
  a {
    color: white;
  }
  width: 100%;
`;
