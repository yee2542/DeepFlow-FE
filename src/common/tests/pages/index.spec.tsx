import { mount, shallow } from 'enzyme';
import React from 'react';
import { Card } from 'antd';
import HomeLayout from 'Modules/home/components/Layouts';
import IconSelector from 'Utils/icon-selector';
import { HeaderWrapper } from 'Modules/home/pages/styled';
import HomePage from 'Modules/home/pages';

describe('Pages/Home test', () => {
  it('Render correctly', () => {
    const wrap = shallow(<HomePage />);
    expect(wrap.html()).toMatchSnapshot();
  });

  it('Home should be exisiting', () => {
    const wrap = shallow(<HomePage />);
    expect(wrap.exists()).toBe(true);
  });

  it('Should contain HomeLayout', () => {
    const wrap = shallow(<HomePage />);
    expect(wrap.exists(HomeLayout)).toBe(true);
  });

  it('Should contain Header', () => {
    const wrap = shallow(<HomePage />);
    expect(wrap.exists(HeaderWrapper)).toBe(true);
  });

  it('Should contain HeaderText', () => {
    const wrap = shallow(<HomePage />);
    expect(wrap.exists(HeaderWrapper)).toBe(true);
  });

  it('Should contain 6 cards 6 icons and 6 headers for card', () => {
    const wrap = mount(<HomePage />);
    expect(wrap.find(Card)).toHaveLength(6);
    expect(wrap.find(IconSelector)).toHaveLength(6);
  });
});
