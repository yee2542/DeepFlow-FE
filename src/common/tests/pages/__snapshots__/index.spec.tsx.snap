// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages/Home test Render correctly 1`] = `"<div><main class=\\"ant-layout-content\\"><div><div class=\\"styled__CanvasWrapper-ha565g-1 faSIVI\\"><div class=\\"styled__HeaderWrapper-ha565g-2 IPEnh\\"><div class=\\"styled__Heading-ha565g-3 ffQEvQ\\">Heading</div><div class=\\"styled__SubHeading-ha565g-4 dolwLX\\">Lorem ipsum..</div><button class=\\"styled__ButtonStyle-czpcv9-0 hSYzlC\\">Explore</button></div></div><div class=\\"ant-row ant-row-space-around\\" style=\\"margin-top:20px;margin-bottom:12px\\"><div style=\\"padding-top:12px;padding-bottom:12px\\" class=\\"ant-col ant-col-21\\"><div class=\\"ant-row ant-row-center\\" style=\\"margin-left:-12px;margin-right:-12px;margin-top:-12px;margin-bottom:12px\\"><div style=\\"padding-left:12px;padding-right:12px;padding-top:12px;padding-bottom:12px\\" class=\\"ant-col ant-col-xs-20 ant-col-md-5 ant-col-lg-7\\"><div style=\\"text-align:center\\" class=\\"ant-card ant-card-bordered\\"><div class=\\"ant-card-body\\"><div class=\\"styled__Icon-ha565g-0 bELiZP\\"><span role=\\"img\\" aria-label=\\"car\\" class=\\"anticon anticon-car\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" data-icon=\\"car\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><path d=\\"M380 704h264c4.4 0 8-3.6 8-8v-84c0-4.4-3.6-8-8-8h-40c-4.4 0-8 3.6-8 8v36H428v-36c0-4.4-3.6-8-8-8h-40c-4.4 0-8 3.6-8 8v84c0 4.4 3.6 8 8 8zm340-123a40 40 0 1080 0 40 40 0 10-80 0zm239-167.6L935.3 372a8 8 0 00-10.9-2.9l-50.7 29.6-78.3-216.2a63.9 63.9 0 00-60.9-44.4H301.2c-34.7 0-65.5 22.4-76.2 55.5l-74.6 205.2-50.8-29.6a8 8 0 00-10.9 2.9L65 413.4c-2.2 3.8-.9 8.6 2.9 10.8l60.4 35.2-14.5 40c-1.2 3.2-1.8 6.6-1.8 10v348.2c0 15.7 11.8 28.4 26.3 28.4h67.6c12.3 0 23-9.3 25.6-22.3l7.7-37.7h545.6l7.7 37.7c2.7 13 13.3 22.3 25.6 22.3h67.6c14.5 0 26.3-12.7 26.3-28.4V509.4c0-3.4-.6-6.8-1.8-10l-14.5-40 60.3-35.2a8 8 0 003-10.8zM840 517v237H184V517l15.6-43h624.8l15.6 43zM292.7 218.1l.5-1.3.4-1.3c1.1-3.3 4.1-5.5 7.6-5.5h427.6l75.4 208H220l72.7-199.9zM224 581a40 40 0 1080 0 40 40 0 10-80 0z\\"></path></svg></span></div><br/><h2>Feature 1</h2><p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus earum iusto</p></div></div></div><div style=\\"padding-left:12px;padding-right:12px;padding-top:12px;padding-bottom:12px\\" class=\\"ant-col ant-col-xs-20 ant-col-md-5 ant-col-lg-7\\"><div style=\\"text-align:center\\" class=\\"ant-card ant-card-bordered\\"><div class=\\"ant-card-body\\"><div class=\\"styled__Icon-ha565g-0 bELiZP\\"><span role=\\"img\\" aria-label=\\"car\\" class=\\"anticon anticon-car\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" data-icon=\\"car\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><path d=\\"M380 704h264c4.4 0 8-3.6 8-8v-84c0-4.4-3.6-8-8-8h-40c-4.4 0-8 3.6-8 8v36H428v-36c0-4.4-3.6-8-8-8h-40c-4.4 0-8 3.6-8 8v84c0 4.4 3.6 8 8 8zm340-123a40 40 0 1080 0 40 40 0 10-80 0zm239-167.6L935.3 372a8 8 0 00-10.9-2.9l-50.7 29.6-78.3-216.2a63.9 63.9 0 00-60.9-44.4H301.2c-34.7 0-65.5 22.4-76.2 55.5l-74.6 205.2-50.8-29.6a8 8 0 00-10.9 2.9L65 413.4c-2.2 3.8-.9 8.6 2.9 10.8l60.4 35.2-14.5 40c-1.2 3.2-1.8 6.6-1.8 10v348.2c0 15.7 11.8 28.4 26.3 28.4h67.6c12.3 0 23-9.3 25.6-22.3l7.7-37.7h545.6l7.7 37.7c2.7 13 13.3 22.3 25.6 22.3h67.6c14.5 0 26.3-12.7 26.3-28.4V509.4c0-3.4-.6-6.8-1.8-10l-14.5-40 60.3-35.2a8 8 0 003-10.8zM840 517v237H184V517l15.6-43h624.8l15.6 43zM292.7 218.1l.5-1.3.4-1.3c1.1-3.3 4.1-5.5 7.6-5.5h427.6l75.4 208H220l72.7-199.9zM224 581a40 40 0 1080 0 40 40 0 10-80 0z\\"></path></svg></span></div><br/><h2>Feature 1</h2><p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus earum iusto</p></div></div></div><div style=\\"padding-left:12px;padding-right:12px;padding-top:12px;padding-bottom:12px\\" class=\\"ant-col ant-col-xs-20 ant-col-md-5 ant-col-lg-7\\"><div style=\\"text-align:center\\" class=\\"ant-card ant-card-bordered\\"><div class=\\"ant-card-body\\"><div class=\\"styled__Icon-ha565g-0 bELiZP\\"><span role=\\"img\\" aria-label=\\"car\\" class=\\"anticon anticon-car\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" data-icon=\\"car\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><path d=\\"M380 704h264c4.4 0 8-3.6 8-8v-84c0-4.4-3.6-8-8-8h-40c-4.4 0-8 3.6-8 8v36H428v-36c0-4.4-3.6-8-8-8h-40c-4.4 0-8 3.6-8 8v84c0 4.4 3.6 8 8 8zm340-123a40 40 0 1080 0 40 40 0 10-80 0zm239-167.6L935.3 372a8 8 0 00-10.9-2.9l-50.7 29.6-78.3-216.2a63.9 63.9 0 00-60.9-44.4H301.2c-34.7 0-65.5 22.4-76.2 55.5l-74.6 205.2-50.8-29.6a8 8 0 00-10.9 2.9L65 413.4c-2.2 3.8-.9 8.6 2.9 10.8l60.4 35.2-14.5 40c-1.2 3.2-1.8 6.6-1.8 10v348.2c0 15.7 11.8 28.4 26.3 28.4h67.6c12.3 0 23-9.3 25.6-22.3l7.7-37.7h545.6l7.7 37.7c2.7 13 13.3 22.3 25.6 22.3h67.6c14.5 0 26.3-12.7 26.3-28.4V509.4c0-3.4-.6-6.8-1.8-10l-14.5-40 60.3-35.2a8 8 0 003-10.8zM840 517v237H184V517l15.6-43h624.8l15.6 43zM292.7 218.1l.5-1.3.4-1.3c1.1-3.3 4.1-5.5 7.6-5.5h427.6l75.4 208H220l72.7-199.9zM224 581a40 40 0 1080 0 40 40 0 10-80 0z\\"></path></svg></span></div><br/><h2>Feature 1</h2><p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus earum iusto</p></div></div></div><div style=\\"padding-left:12px;padding-right:12px;padding-top:12px;padding-bottom:12px\\" class=\\"ant-col ant-col-xs-20 ant-col-md-5 ant-col-lg-7\\"><div style=\\"text-align:center\\" class=\\"ant-card ant-card-bordered\\"><div class=\\"ant-card-body\\"><div class=\\"styled__Icon-ha565g-0 bELiZP\\"><span role=\\"img\\" aria-label=\\"car\\" class=\\"anticon anticon-car\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" data-icon=\\"car\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><path d=\\"M380 704h264c4.4 0 8-3.6 8-8v-84c0-4.4-3.6-8-8-8h-40c-4.4 0-8 3.6-8 8v36H428v-36c0-4.4-3.6-8-8-8h-40c-4.4 0-8 3.6-8 8v84c0 4.4 3.6 8 8 8zm340-123a40 40 0 1080 0 40 40 0 10-80 0zm239-167.6L935.3 372a8 8 0 00-10.9-2.9l-50.7 29.6-78.3-216.2a63.9 63.9 0 00-60.9-44.4H301.2c-34.7 0-65.5 22.4-76.2 55.5l-74.6 205.2-50.8-29.6a8 8 0 00-10.9 2.9L65 413.4c-2.2 3.8-.9 8.6 2.9 10.8l60.4 35.2-14.5 40c-1.2 3.2-1.8 6.6-1.8 10v348.2c0 15.7 11.8 28.4 26.3 28.4h67.6c12.3 0 23-9.3 25.6-22.3l7.7-37.7h545.6l7.7 37.7c2.7 13 13.3 22.3 25.6 22.3h67.6c14.5 0 26.3-12.7 26.3-28.4V509.4c0-3.4-.6-6.8-1.8-10l-14.5-40 60.3-35.2a8 8 0 003-10.8zM840 517v237H184V517l15.6-43h624.8l15.6 43zM292.7 218.1l.5-1.3.4-1.3c1.1-3.3 4.1-5.5 7.6-5.5h427.6l75.4 208H220l72.7-199.9zM224 581a40 40 0 1080 0 40 40 0 10-80 0z\\"></path></svg></span></div><br/><h2>Feature 1</h2><p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus earum iusto</p></div></div></div><div style=\\"padding-left:12px;padding-right:12px;padding-top:12px;padding-bottom:12px\\" class=\\"ant-col ant-col-xs-20 ant-col-md-5 ant-col-lg-7\\"><div style=\\"text-align:center\\" class=\\"ant-card ant-card-bordered\\"><div class=\\"ant-card-body\\"><div class=\\"styled__Icon-ha565g-0 bELiZP\\"><span role=\\"img\\" aria-label=\\"car\\" class=\\"anticon anticon-car\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" data-icon=\\"car\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><path d=\\"M380 704h264c4.4 0 8-3.6 8-8v-84c0-4.4-3.6-8-8-8h-40c-4.4 0-8 3.6-8 8v36H428v-36c0-4.4-3.6-8-8-8h-40c-4.4 0-8 3.6-8 8v84c0 4.4 3.6 8 8 8zm340-123a40 40 0 1080 0 40 40 0 10-80 0zm239-167.6L935.3 372a8 8 0 00-10.9-2.9l-50.7 29.6-78.3-216.2a63.9 63.9 0 00-60.9-44.4H301.2c-34.7 0-65.5 22.4-76.2 55.5l-74.6 205.2-50.8-29.6a8 8 0 00-10.9 2.9L65 413.4c-2.2 3.8-.9 8.6 2.9 10.8l60.4 35.2-14.5 40c-1.2 3.2-1.8 6.6-1.8 10v348.2c0 15.7 11.8 28.4 26.3 28.4h67.6c12.3 0 23-9.3 25.6-22.3l7.7-37.7h545.6l7.7 37.7c2.7 13 13.3 22.3 25.6 22.3h67.6c14.5 0 26.3-12.7 26.3-28.4V509.4c0-3.4-.6-6.8-1.8-10l-14.5-40 60.3-35.2a8 8 0 003-10.8zM840 517v237H184V517l15.6-43h624.8l15.6 43zM292.7 218.1l.5-1.3.4-1.3c1.1-3.3 4.1-5.5 7.6-5.5h427.6l75.4 208H220l72.7-199.9zM224 581a40 40 0 1080 0 40 40 0 10-80 0z\\"></path></svg></span></div><br/><h2>Feature 1</h2><p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus earum iusto</p></div></div></div><div style=\\"padding-left:12px;padding-right:12px;padding-top:12px;padding-bottom:12px\\" class=\\"ant-col ant-col-xs-20 ant-col-md-5 ant-col-lg-7\\"><div style=\\"text-align:center\\" class=\\"ant-card ant-card-bordered\\"><div class=\\"ant-card-body\\"><div class=\\"styled__Icon-ha565g-0 bELiZP\\"><span role=\\"img\\" aria-label=\\"car\\" class=\\"anticon anticon-car\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" data-icon=\\"car\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><path d=\\"M380 704h264c4.4 0 8-3.6 8-8v-84c0-4.4-3.6-8-8-8h-40c-4.4 0-8 3.6-8 8v36H428v-36c0-4.4-3.6-8-8-8h-40c-4.4 0-8 3.6-8 8v84c0 4.4 3.6 8 8 8zm340-123a40 40 0 1080 0 40 40 0 10-80 0zm239-167.6L935.3 372a8 8 0 00-10.9-2.9l-50.7 29.6-78.3-216.2a63.9 63.9 0 00-60.9-44.4H301.2c-34.7 0-65.5 22.4-76.2 55.5l-74.6 205.2-50.8-29.6a8 8 0 00-10.9 2.9L65 413.4c-2.2 3.8-.9 8.6 2.9 10.8l60.4 35.2-14.5 40c-1.2 3.2-1.8 6.6-1.8 10v348.2c0 15.7 11.8 28.4 26.3 28.4h67.6c12.3 0 23-9.3 25.6-22.3l7.7-37.7h545.6l7.7 37.7c2.7 13 13.3 22.3 25.6 22.3h67.6c14.5 0 26.3-12.7 26.3-28.4V509.4c0-3.4-.6-6.8-1.8-10l-14.5-40 60.3-35.2a8 8 0 003-10.8zM840 517v237H184V517l15.6-43h624.8l15.6 43zM292.7 218.1l.5-1.3.4-1.3c1.1-3.3 4.1-5.5 7.6-5.5h427.6l75.4 208H220l72.7-199.9zM224 581a40 40 0 1080 0 40 40 0 10-80 0z\\"></path></svg></span></div><br/><h2>Feature 1</h2><p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus earum iusto</p></div></div></div></div></div></div></div></main><footer class=\\"ant-layout-footer\\">Footer</footer></div>"`;
